.github/ISSUE_TEMPLATE/01_doc_request.yml

```
@@ -0,0 +1,29 @@
+name: 📚 Documentation update request
+description: Suggest a change to existing Redpanda documentation.
+labels: [doc request]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Help us improve Redpanda documentation! You can search [open doc issues](https://github.com/redpanda-data/documentation/issues), and ask for guidance on our [community Slack docs channel](https://redpanda.com/slack).
+
+        Create issues for doc bugs, improvements, or content gaps. If you need to request a new feature, consider using the New Feature template.
+  - type: textarea
+    attributes:
+      label: Issue Description
+      description: Describe how the current doc is incorrect, incomplete, or unclear, and describe how to fix it. Include as much information as possible.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Updates to existing documentation
+      description: Include the URL of the documentation page that requires an update and any links to additional details or Slack conversations about the issue.
+    validations:
+      required: true  
+  - type: textarea
+    attributes:
+      label: Link to Redpanda Community Slack conversation
```

```suggestion
      label: Link to Redpanda Slack conversation
```
suggest removing Community, because we don't want internal Redpanda folks to think this form is only for issues coming from external users